src/components/inputs/TextField.js

```diff
@@ -642,6 +679,19 @@ function createStyles({placeholderTextColor, centered, multiline}) {
     label: {
       ...LABEL_TYPOGRAPHY,
       height: LABEL_TYPOGRAPHY.lineHeight
+    },
+    rightIcon: {
+      alignSelf: 'center', 
+      marginLeft: 8
+    },
+    rightButton: {
+      position: 'absolute',
+      right: 3,
+      alignSelf: 'center'
+    },
+    rightButtonImage: {
```

Comment:
I wouldn't transform the size of the image, it might break the image proportion.

Reply:
It is not designed to contain just any image, the point is to have an icon of size 24 there. Or not?

Reply:
but it's open source, people can pass whatever.